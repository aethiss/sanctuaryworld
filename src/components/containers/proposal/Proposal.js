import React from 'react'
// import PropTypes from 'prop-types'

// Components
import BodyContainer from '../../commons/wrapper/BodyContainerResponsive'

const Proposal = () => {
  return (
    <BodyContainer>
      <div>Proposal !</div>
    </BodyContainer>
  )
}

export default Proposal
