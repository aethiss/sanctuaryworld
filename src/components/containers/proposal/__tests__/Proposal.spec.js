import React from 'react'
import TestRenderer from 'react-test-renderer'
import Proposal from '../Proposal'


describe('Proposal', () => {
  describe('Render', () => {
    it('[Proposal] : Render default', () => {
      const testRenderer = TestRenderer.create(<Proposal proposal='world' />)
      expect(testRenderer).toMatchSnapshot()
    })
  })
})
