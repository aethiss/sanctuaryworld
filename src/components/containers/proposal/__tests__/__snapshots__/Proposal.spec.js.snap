// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Proposal Render [Proposal] : Render default 1`] = `
<div
  className="makeStyles-bodyWrapper-1"
>
  <div>
    Proposal !
  </div>
</div>
`;
